iron-component-page.html

```diff
@@ -81,7 +81,7 @@
     <div columns>
 
       <div docs>
-        <iron-doc-viewer src="{{_docUrl}}"></iron-doc-viewer>
+        <iron-doc-viewer nodemo src="{{_docUrl}}"></iron-doc-viewer>
```

Comment:
It should be `no-demo`.
https://github.com/PolymerElements/iron-doc-viewer/blob/master/iron-doc-viewer.html#L130